libs/feature/editor/src/lib/fields.config.ts

```diff
@@ -1,71 +1,198 @@
 import { marker } from '@biesbjerg/ngx-translate-extract-marker'
-import { EditorFieldsConfig } from './models/fields.model'
-
-export const DEFAULT_FIELDS: EditorFieldsConfig = [
-  {
-    model: 'title',
-    formFieldConfig: {
-      labelKey: 'Metadata title',
-      type: 'text',
-    },
+import {
+  EditorField,
+  EditorFieldsConfig,
+  EditorFieldSection,
+} from './models/fields.model'
+
+/**
+ * This file contains the configuration of the fields that will be displayed in the editor.
+ * To add a new field, you need to create a new EditorField object in the fields part of this file.
+ * Then add it to the corresponding section in the sections part of this file.
+ * Finally, add the section to the corresponding page in the pages part of this file.
+ */
+
+/************************************************************
+ ***************           FIELDS           *****************
+ ************************************************************
+ */
+
+export const RECORD_LICENSE_FIELD: EditorField = {
+  model: 'licenses',
+  formFieldConfig: {
+    labelKey: marker('editor.record.form.license'),
+    type: 'list',
```

Comment:
Would it cost you much to remove this `type` property everywhere? It's not used anymore, and your PR already touches the needed files.